src/write.rs

```diff
@@ -431,14 +431,30 @@ where
     /// Remove the given value from the value-bag of the given key.
     ///
     /// The updated value-bag will only be visible to readers after the next call to `refresh()`.
+    #[deprecated(since = "11.0.0", note = "Renamed to remove_value")]
     pub fn remove(&mut self, k: K, v: V) -> &mut Self {
+        self.remove_value(k, v)
+    }
+
+    /// Remove the given value from the value-bag of the given key.
+    ///
+    /// The updated value-bag will only be visible to readers after the next call to `refresh()`.
+    pub fn remove_value(&mut self, k: K, v: V) -> &mut Self {
         self.add_op(Operation::Remove(k, v))
     }
 
     /// Remove the value-bag for the given key.
     ///
     /// The value-bag will only disappear from readers after the next call to `refresh()`.
+    #[deprecated(since = "11.0.0", note = "Renamed to remove_entry")]
     pub fn empty(&mut self, k: K) -> &mut Self {
+        self.remove_entry(k)
+    }
+
+    /// Remove the value-bag for the given key.
+    ///
+    /// The value-bag will only disappear from readers after the next call to `refresh()`.
+    pub fn remove_entry(&mut self, k: K) -> &mut Self {
```

Comment:
Oh, also, I later discovered that [`remove_entry`](https://doc.rust-lang.org/beta/std/collections/struct.HashMap.html#method.remove_entry) also exists for `std::collections::HashMap` so maybe we don't want to use this because this doesn't use the Entry API (yet?). Maybe this should be `remove_key` or `remove_bag` or something?

Reply:
It was precisely because of `HashMap::remove_entry` that I suggested the name :p The fact that it includes the key as well gives more of the flavor of "this removes the whole thing".

Reply:
Haha okay! Yeah I don't think we need more words to describe "removes the whole thing" (as the expected API is probably just "remove") but I didn't know if we should reach for "not `_entry`" because one might expect `remove_entry` should return `Option<(K, V)>` and this doesn't.

Reply:
Nah, I think it's probably fine the way it is. People already realize pretty quickly that `evmap` doesn't have the same interface as `HashMap`.